api/types/types.go

```diff
@@ -170,6 +170,7 @@ type Info struct {
 	MemTotal           int64
 	GenericResources   []swarm.GenericResource
 	DockerRootDir      string
+	DNSDefaults        DNSInfo
```

Comment:
Naming this `xxDefaults` as I've seen there's confusion about the `--dns` configuration in general (i.e., this is what's used as a default for _containers_ that are started, but it's not the DNS configuration for the daemon process _itself_.